### PR TITLE
Update to Go 1.19

### PR DESCRIPTION
**Pull request by jan-dubsky**
The Go1.12 is quite limiting. It's time to use newer Go version.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,3 @@
 .idea
 .debug
-go.sum
```
Comment (jan-dubsky) [inline .gitignore:3]: According to Go module guidelines, the go.sum should be committed in the repository.

File: .gitignore [hunk 2]
```diff
@@ -1,3 +1,3 @@
...
\ No newline at end of file
+coverage.txt
```
Comment (jan-dubsky) [inline .gitignore:3]: File produced by `make test`

**Comment by negasus** [comment]
Can you please resolve conflicts?

**Comment by jan-dubsky** [comment]
> Can you please resolve conflicts?

Resolved.

**Pull request merged**